packages/@okta/vuepress-site/docs/reference/csi-delauth-hook/index.md

```diff
@@ -0,0 +1,541 @@
+
+| Property | Description                                           | Data Type       |
+|----------|-------------------------------------------------------|-----------------|
+| type     | One of the [supported commands](#supported-commands). | String          |
+| value    | The parameter to pass to the command.                 | [value](#value) |
+
+##### Supported Commands
+
+The following command is supported for the delegated authentication request:
+
+| Command                         | Description                                                                                                              |
+|---------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| com.okta.action.update          | Specify the result of the authentication, either success or one of the causes of failure. |
+
+
+##### value
+
+The `value` object is the parameter to pass to the command. For the `com.okta.action.update` command, `value` should be an object containing a `credential` property set to one of the following values:
+
+- `VERIFIED`: The user is authenticated successfully
+- `UNVERIFIED`: The user fails to authenticate due to invalid credential
+- `ACCOUNT_LOCKED`: The user account is locked
+- `ACCOUNTOUN_DISABLED`: The user account is disabled
+- `PASSWORD_EXPIRED`: The user password has expired
+- `UNKNOWN_USER`: The user does not exist in the source
+
+For example, to indicate the user authencates successfully you would return the following:
+
+```json
+{
+  "commands": [{
+    "type": "com.okta.user.profile.update",
+    "value": {
+      "credential": "VERIFIED"
+    }
+  }]
+}
+```
+
+Another example for the situation when the user account fails authentication due to account being disabled:
+
+```json
+{
+  "commands": [{
+    "type": "com.okta.action.update",
+    "value": {
+      "credential": "ACCOUNT_DISABLED"
+    }
+  }]
+}
+```
+
+#### error
+
+When you return an error object, it should contain an `errorSummary` sub-object:
+
+| Property     | Description                          | Data Type |
+|--------------|--------------------------------------|-----------|
+| errorSummary | Human-readable summary of the error. | String    |
+
+Returning an error object will cause Okta to record a failure event in the Okta System Log. The string you supplied in the `errorSummary` property of the `error` object will be recorded in the System Log event.
+
+### Sample JSON Payload of Request from Okta to Your External Service
+
+```json
+{
+  "eventId": "3o5iAzq1SmOGmmssqDyzeM",
+  "eventTime": "2020-12-10T23:10:33.000Z",
+  "eventType": "com.okta.custom.source.delegated.authentication",
+  "eventTypeVersion": "1.0",
+  "contentType": "application/json",
+  "cloudEventVersion": "0.1",
+  "requestType": "user.authenticate",
+  "source": "https://${yourOktaDomain}/api/v1/inlineHooks/cal11h7VDs8LC4DAi0g4",
```

Comment:
```suggestion
  "source": "https://{yourOktaDomain}/api/v1/inlineHooks/cal11h7VDs8LC4DAi0g4",
```

Reply:
Style issue we're trying to fix up across dev docs: removing $ from code samples that are not requesting input.